src/routes_api.py

```diff
@@ -21,8 +21,9 @@ def admin_update_roster(netid, cid):
             return util.success("Successfully updated roster.", HTTPStatus.OK)
 
         @blueprint.route("/api/<cid>/<aid>/trigger_scheduled_run/<scheduled_run_id>", methods=["POST"])
-        def trigger_scheduled_run(cid, aid, scheduled_run_id):
-            # TODO add system token authentication
+        @auth.require_token_auth
+        @auth.require_admin_status
```

Comment:
Can we change this to use the system token instead? 
https://github.com/illinois-cs241/broadway-on-demand/blob/master/sample_config.py#L29